File: Java_ChildrenCompetition/src/main/java/repository/TestParticipantRelationRepository.java
package repository;

import model.TestParticipantRelation;
import model.Tuple;

public class TestParticipantRelationRepository extends AbstractRepository<Tuple<Integer, Integer>, TestParticipantRelation>{
    public TestParticipantRelationRepository(Validator<TestParticipantRelation> validator){
        super(validator);
    }
}

File: CSharp_ChildrenCompetition/CSharp_ChildrenCompetition/CSharp_ChildrenCompetition/repository/TestParticipantRelationRepository.cs
using System;
using CSharp_ChildrenCompetition.model;

namespace CSharp_ChildrenCompetition.repository
{
    public class TestParticipantRelationRepository : AbstractRepository<Tuple<int, int>, TestParticipantRelation>
    {
        public TestParticipantRelationRepository(Validator<TestParticipantRelation> validator) : base(validator)
        {
            
        }
    }
}